airbyte-workers/src/main/java/io/airbyte/workers/temporal/scheduling/ConnectionManagerWorkflowImpl.java

```diff
@@ -102,6 +103,9 @@ public class ConnectionManagerWorkflowImpl implements ConnectionManagerWorkflow
   private static final String RENAME_ATTEMPT_ID_TO_NUMBER_TAG = "rename_attempt_id_to_number";
   private static final int RENAME_ATTEMPT_ID_TO_NUMBER_CURRENT_VERSION = 1;
 
+  private static final String CHECK_PREVIOUS_JOB_OUTPUT_TAG = "rename_attempt_id_to_number";
```

Comment:
This tag need to be unique. Currently it is the same than `RENAME_ATTEMPT_ID_TO_NUMBER_TAG`

Reply:
oh, that's my inattention. Thanks for noticing this